_pages/speakers.md

```diff
@@ -7,7 +7,32 @@ permalink: /speakers/
 
 <div class="pretty-links">
 
-We'll be putting out a call for talk proposals in early fall, and will be putting any updates here!
+## Keynotes
+
+### Julia Evans
+
+![](/assets/img/speakers/julia.jpg)
+
+She's great. Here are some words from her site as a placeholder: I live in Montreal. I work at Stripe, on infrastructure. I read all my email and try to respond. I sometimes give talks. I have one main opinion about programming and it’s – understanding the underlying systems you use (the kernel, the operating system, the network layers, your database, HTTP, whatever you’re running on top of) is essential if you want to do amazing work and be able to fix hard problems. It’s served me well so far.
+
+### Safia Abdalla
+
+![](https://pbs.twimg.com/profile_images/871404250060009472/8vegZkvO_400x400.jpg)
+
+This is all temp because Safia hasn't given us the deets yet. Her twitter bio is: I'm building http://zarf.co  and sharing the story on a podcast at http://log.zarf.co . I write a lot, run a lot, code a lot, and dream big (a lot). She's at a conference right now. This is maybe how long her bio will be?
+
+## Invited lightning talk speakers
+
+### Dinah Davis
+
+![](/assets/img/speakers/dinah.jpg)
+
+Dinah Davis is the Director of R&D at Arctic Wolf Networks, responsible for the development of the AWN CyberSOC product. Dinah holds an M. Math in cryptography from the University of Waterloo. She has over 16 years of experience in the tech industry with time spent as a Security Software Developer at the Communications Security Establishment Canada and BlackBerry, and as a manager at TrustWave and D2l. Dinah is also the founder of Code Like a Girl, a publication with over 24 000 subscribers and a vibrant online community.
```

Comment:
Is this the version of Dinah's bio you want to use? Just so I can get her to check it over